src/libs/actions/IOU.ts

```diff
@@ -8404,8 +8404,7 @@ function canSubmitReport(
     const isAdmin = policy?.role === CONST.POLICY.ROLE.ADMIN;
     const transactionIDList = transactions.map((transaction) => transaction.transactionID);
     const hasAllPendingRTERViolations = allHavePendingRTERViolation(transactionIDList, allViolations);
-    const hasBrokenConnectionViolation = shouldShowBrokenConnectionViolationForMultipleTransactions(transactionIDList, report, policy, allViolations);
-
+    const hasAllRTEViolations = allHaveRTERViolation(transactionIDList, allViolations);
```

Comment:
Can you please make sure to invert this: ie just check if there is at least one transaction  that does not have the RTER violation? that should work the same right? and should theoretically go faster
```suggestion
    const hasAllRTERViolations = allHaveRTERViolation(transactionIDList, allViolations);
```